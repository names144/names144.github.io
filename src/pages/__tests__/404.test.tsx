import * as React from 'react';
import renderer from 'react-test-renderer';

import NotFoundPage from '../404';

describe('NotFoundPage', () => {
  test('Should render', () => {
    const tree = renderer
      .create(<NotFoundPage />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
