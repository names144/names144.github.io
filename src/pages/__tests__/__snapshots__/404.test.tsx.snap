// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage Should render 1`] = `
<div>
  <mock-seo
    title="404"
  />
  <h1>
    404: Not Found
  </h1>
</div>
`;
